stdlib/experimental/http/http_experimental.go

```diff
@@ -31,7 +27,7 @@ var get = values.NewFunction(
 			"timeout": semantic.Duration,
 		},
 		Required: []string{"url"},
-		Return:   semantic.NewObjectPolyType(map[string]semantic.PolyType{"statusCode": semantic.Int, "headers": semantic.Object, "body": semantic.Bytes}, semantic.LabelSet{"status", "headers", "body"}, nil),
+		Return:   semantic.NewObjectPolyType(map[string]semantic.PolyType{"statusCode": semantic.Int, "headers": semantic.Object, "body": semantic.Bytes}, semantic.LabelSet{"statusCode", "headers", "body"}, nil),
```

Comment:
Why does the `status` need to change to `statusCode`?

Reply:
It looks like the previous code was an error. The poly type has `statusCode` but the label set had `status`.